angular-spa-ui/js/common/bootstrap.js

```diff
@@ -0,0 +1,107 @@
+"use strict";
+
+module.exports = function ($rootScope, promises, configService, appLocalStorage, commonService, getTemplate, $q) {
+
+    var config = configService.getConfig('appConfig');
+    var paths = config.paths;
+    var userData = appLocalStorage.getItem('userData') || {};
+
+    userData.userId = GLOBAL_USER_ID;
+    paths.userPath += GLOBAL_USER_ID;
+
+    return {
+        checkAllSteps: function () {
+            var deferred = $q.defer();
+            userData = {};
+            config = configService.getConfig('appConfig');
+            paths = config.paths;
+            paths.userPath += GLOBAL_USER_ID;
+
+            promises.getAsyncData('GET', paths.userPath)
+                .then(
+                    function (data) {
+                        if (data.status === 'error') {
+                            var errObject = {
+                                code: data.error.name,
```

Comment:
<pre>Do
  you
    like
      spaghetti
        code?
          I sure
            nobody
              likes
                to support
                  spaghetti
                    code
                      because
                        it's not cool</pre>